src/docs/config/overview.md

```diff
@@ -54,10 +54,17 @@ enableCache: true
 
 The global script runs once before your library/app loads, so you can do things like setting up a connection to an external service or configuring a library you are using.
 
-The code to be executed should be placed within a default function that is exported by the global script. Ensure all of the code in the global script is wrapped in the function that is exported.
+The code to be executed should be placed within a default function that is exported by the global script. Ensure all of the code in the global script is wrapped in the function that is exported:
+
+```javascript
```

Comment:
Is it still that highlighting only works if this is set to `typescript`?

Reply:
Not sure, it looks ok in the preview here.